bool check_status
---
bool status_record_bag